.github/workflows/ai-codereview.yml

```diff
@@ -0,0 +1,87 @@
+name: AI Codereview
+on:
+  pull_request:
```

Comment:
@eromanova could you elaborate on this, we should set `pull_request_target` here as it is in the other jobs, right?